### ISSUE TITLE
TypeError: Cannot read properties of undefined (reading 'split') when using hardhat compile and import "@openzeppelin/hardhat-upgrades"

### ISSUE THREAD

**Issue by jesusholoo** [OpenZeppelin/openzeppelin-upgrades#891]
The following error happens when executing `hardhat compile` :
```
TypeError: Cannot read properties of undefined (reading 'split')
    at getPositions (/home/user/zkevm-contracts/node_modules/@openzeppelin/upgrades-core/src/utils/make-namespaced.ts:117:31)
    at makeDelete (/home/user/zkevm-contracts/node_modules/@openzeppelin/upgrades-core/src/utils/make-namespaced.ts:141:19)
    at makeNamespacedInput (/home/user/zkevm-contracts/node_modules/@openzeppelin/upgrades-core/src/utils/make-namespaced.ts:68:14)
    at OverriddenTaskDefinition._action (/home/user/zkevm-contracts/node_modules/@openzeppelin/hardhat-upgrades/src/index.ts:111:21)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async Environment._runTaskDefinition (/home/user/zkevm-contracts/node_modules/hardhat/src/internal/core/runtime-environment.ts:358:14)
    at async Environment.run (/home/user/zkevm-contracts/node_modules/hardhat/src/internal/core/runtime-environment.ts:191:14)
    at async SimpleTaskDefinition.action (/home/user/zkevm-contracts/node_modules/hardhat/src/builtin-tasks/compile.ts:1009:37)
    at async Environment._runTaskDefinition (/home/user/zkevm-contracts/node_modules/hardhat/src/internal/core/runtime-environment.ts:358:14)
    at async Environment.run (/home/user/zkevm-contra
```
If i remove the ` import "@openzeppelin/hardhat-upgrades";` form the `hardhat.config.ts` it does work 

After some painfull debugging i found out that the contract that causes the issue is:  https://github.com/0xPolygonHermez/zkevm-contracts/blob/9e55f5705f567a203d72ac861ad80033b0a5490a/contracts/mocks/Uni.sol
Which should be exactly the UNI token copied from etherscan ( which i use for testing)

I manage to reproduce the issue in another computer, you can download this repo:
https://github.com/0xPolygonHermez/zkevm-contracts/tree/9e55f5705f567a203d72ac861ad80033b0a5490a

And execute the following command to trigger the error:
`npm i`
`npx hardhat compile`

I'ts enough to just compile the `contract Uni` to trigger the error.
Node version: v18.16.1
Npm version: 9.5.1

**Comment by ericglau** [comment]
Thank you for reporting this and for the detailed steps to reproduce the issue.  This will be fixed in #892 and we will put out an updated release as soon as that is ready.

**Comment by invocamanman** [comment]
thank youu! ^^ was a real quick fix hehe :D 
It was a painful debugging but glad to help^^
